docs/section22/Rule22-11.md

```diff
@@ -0,0 +1,46 @@
+
+# CHW&CW - Rule 22-11  
+
+**Rule ID:** 22-11  
+**Rule Description:** For Baseline chilled-water system that does not use purchased chilled water, variable-flow secondary pump shall be modeled as 13W/gpm at design conditions.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**Appendix G Section Reference:** Section G3.1.3.10 Chilled-water supply temperature reset (System 7, 8, 11, 12 and 13)  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 11, 12, or 13.
+2. B-RMR is modeled with at least one chilled water system that does not use purchased chilled water.
+
```

Comment:
I'd add that rule 22-7 passes (CHW plant is correctly modeled with a secondary loop.)

Reply:
if we add 22-7 pass applicability check, then we can remove #1, as 22-7 already verifies #1